lib/util.js

```diff
@@ -527,6 +527,15 @@ const util = {
   },
 
   gclientSync: (reset = false, options = {}) => {
+    if (fs.existsSync(config.srcDir)) {
+      console.log('setting src remote `origin` to ' + config.chromiumRepo)
+      util.run('git', ['-C', config.srcDir, 'remote', 'remove', 'origin'], options)
+      util.run('git', ['-C', config.srcDir, 'remote', 'add', 'origin', config.chromiumRepo], options)
+      console.log('setting src/brave remote `origin` to ' + config.braveCoreRepo)
```

Comment:
CI build failed here
need to add the check for braveCoreDir: `if (fs.existsSync(config.braveCoreDir))`